multus/multus.go

```diff
@@ -359,6 +359,11 @@ func delPlugins(exec invoke.Exec, argIfname string, delegates []*types.DelegateN
 	return nil
 }
 
+func cmdErr(k8sArgs *types.K8sArgs, format string, args ...interface{}) error {
+	msg := fmt.Sprintf(format, args...)
+	return logging.Errorf("Multus: [%s/%s]: %s", k8sArgs.K8S_POD_NAMESPACE, k8sArgs.K8S_POD_NAME, msg)
```

Comment:
Could you please add alternative return in case of k8sArgs==nil for future?

Reply:
Done!